src/fixml/modules/template.py

```diff
@@ -0,0 +1,35 @@
+from jinja2 import Environment, PackageLoader, Template, meta
+
+
+class TemplateLoader:
+    """Static wrapper class for setting states for Jinja2's Environment."""
+
+    template_exts = ["jinja", "j2"]
+    env = Environment(
+        loader=PackageLoader("test_creation.data", "templates")
+    )
+    template_aliases = {
+        "evaluation": "eval_report.md.jinja",
+        "checklist": "checklist.md.jinja",
+    }
+
+    def __new__(cls):
+        raise TypeError("This static class cannot be instantiated.")
+
+    @classmethod
+    def load(cls, template_name: str) -> Template:
+        candidates = [template_name, cls.template_aliases.get(template_name)]
```

Comment:
what is the expected behaviour if there are multiple non-None candidates?

Reply:
The first non-None candidate will always be loaded. For example:
```
>>> TemplateLoader.env.select_template(["eval_report.md.jinja", "checklist.md.jinja"])
<Template 'eval_report.md.jinja'>
```
See docs on [`jinja2.Environment.select_template()`](https://jinja.palletsprojects.com/en/3.0.x/api/#jinja2.Environment.select_template)

Reply:
in this case, do you think if it's better to enforce one template name like this?
```
def load(cls, template_name: str) -> Template:
    alias = cls.template_aliases.get(template_name)
    if alias:
        return cls.env.get_template(alias)
    else:
        return cls.env.get_template(template_name)
```

Reply:
I would rather the actual template name to take precedence in the case of an alias having the same name with a template in the data folder. In this case, we will have to write a `try/except` loop to handle this e.g. try with actual name first, then try alias if it fails to load, which is not ideal.

Reply:
Do you have few minutes for a quick call? I think that's faster hahaha

Reply:
yeah sure

Reply:
Summary: In the case of User specified template_name equal to one of the cls.template_aliases.keys(), we always employ the user specified. That's why we write in this way